server/server.go

```diff
@@ -487,7 +545,7 @@ func (s *Server) leadershipAcquired() error {
 	s.leaderSub = sub
 
 	atomic.StoreInt64(&(s.getRaft().leader), 1)
-	return nil
```

Comment:
Why remove this?

Reply:
This looks like a merge that has somehow failed. Should be fixed in 0cdb37a.